.github/workflows/generate_ebooks.yml

```diff
@@ -0,0 +1,77 @@
+######################################
+## Custom Web Almanac GitHub action ##
+######################################
+#
+# Generate the ebooks when this GitHub Action is run manually
+# This should be run everytime before release
+#
+name: Generate Ebooks
+
+env:
+  # Update periodically from https://www.princexml.com/latest/
+  PRINCE_PACKAGE: 'prince_20200728-1_ubuntu20.04_amd64.deb'
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    # Prince needs ubuntu 20 and ubuntu-latest is only on 18
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+    - name: Setup Node.js for use with actions
+      uses: actions/setup-node@v1.1.0
+      with:
+        version:  12.x
+    # Install Python 3.7 as used by Google Cloud Platform
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install Python requirements
+      run: |
+        echo "Installing Python requirements"
+        python -m pip install --upgrade pip
+        cd src
+        pip install -r requirements.txt
+    - name: Install node modules
+      run: |
+        echo "Running npm install"
+        cd src
+        npm install
+    - name: Install PrinceXML
+      run: |
+        echo "Installing PrinceXML"
+        cd /tmp
+        sudo apt-get install -y aptitude
+        sudo aptitude install -y gdebi
```

Comment:
Is it necessary to install `aptitude` first and then use it? Why can't we just use `apt`?

Reply:
That's the recommended way to install PrinceXML: https://www.princexml.com/doc/installing/#installing-prince-on-linux-freebsd

Reply:
Honestly speaking, I would not go through this lengthy route. I would dome something like the following that will take care of any dependencies as well:

```
# Set the package file name as ENV in the beginning
PRINCE_PACKAGE=prince_13.5-1_ubuntu18.04_amd64.deb

cd /tmp
wget https://www.princexml.com/download/prince_13.5-1_ubuntu18.04_amd64.deb
DEBIAN_FRONTEND=noninteractive sudo apt install -y /tmp/${{ env.PRINCE_PACKAGE }}
```

Reply:
This seems to work too - so switched to that as a lot quicker. Can switch back if it proves problematic. Thanks!